internal/oem/oem.go

```diff
@@ -103,10 +105,129 @@ func init() {
-				}},
+				Units: []types.SystemdUnit{
+					{
+						Name:   "coreos-metadata-sshkeys@.service",
+						Enable: true,
+					},
+					{
+						Name:   "google-accounts-manager.service",
+						Enable: true,
+						Contents: `
+[Unit]
+Description=Google Compute Engine User Accounts Manager Daemon
+After=network.target
+Before=sshd.service
+Requires=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/share/oem/python/bin/python2.7 /usr/share/oem/google-compute-daemon/manage_accounts.py
+
+[Install]
+WantedBy=multi-user.target
+`,
+					},
+					{
+						Name:   "google-address-manager.service",
+						Enable: true,
+						Contents: `
+[Unit]
+Description=Google Compute Engine Address Manager Daemon
+After=network.target
+Requires=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/share/oem/python/bin/python2.7 /usr/share/oem/google-compute-daemon/manage_addresses.py
+
+[Install]
+WantedBy=multi-user.target
+`,
+					},
+					{
+						Name:   "google-clock-sync-manager.service",
+						Enable: true,
+						Contents: `
+[Unit]
+Description=Google Compute Engine Clock Sync Daemon
+After=network.target
+Requires=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/share/oem/python/bin/python2.7 /usr/share/oem/google-compute-daemon/manage_clock_sync.py
+
+[Install]
+WantedBy=multi-user.target
+`,
+					},
+					{
+						Name:   "google-startup-scripts-onboot.service",
+						Enable: true,
+						Contents: `
+[Unit]
+Description=Google Compute Engine Startup Scripts
+After=network.target
+After=local-fs.target
+Before=sshd.service
+Requires=network.target
+Requires=local-fs.target
+
+[Service]
+Type=oneshot
+Environment=GOOGLE_STARTUP_SCRIPTS_PREFIX=/usr/share/oem/google-startup-scripts
+ExecStart=/usr/share/oem/google-startup-scripts/onboot
+
+[Install]
+WantedBy=multi-user.target
+`,
+					},
+					{
+						Name:   "google-startup-scripts.service",
+						Enable: true,
+						Contents: `
+[Unit]
+Description=Google Compute Engine user startup scripts
+After=network.target
+After=google-startup-scripts-onboot.service
+Requires=google-startup-scripts-onboot.service
+Requires=network.target
+
+[Service]
+ExecStart=/usr/share/oem/google-startup-scripts/run-startup-scripts
+Type=idle
+
+[Install]
+WantedBy=multi-user.target
+`,
+					},
+					{
+						Name:   "oem-cloudinit.service",
+						Enable: true,
+						Contents: `
+[Unit]
+Requires=coreos-setup-environment.service
+After=coreos-setup-environment.service
+
+[Service]
+Type=oneshot
+EnvironmentFile=/etc/environment
+ExecStart=/usr/share/oem/bin/gce-coreos-cloudinit
+`,
+					},
+				},
+			},
+			Storage: types.Storage{
+				Files: []types.File{
+					{
+						Filesystem: "root",
+						Path:       "/etc/hosts",
+						Mode:       292,
+						Contents:   contentsFromString("169.254.169.254 metadata"),
+					},
+					oemId("ID=gce\nNAME=Google Compute Engine\nHOME_URL=https://cloud.google.com/products/compute-engine/\nBUG_REPORT_URL=https://github.com/coreos/bugs/issues"),
```

Comment:
The oem-release includes the oem version which should reflect what is installed in /usr/share/oem